submissions/gorodiskiy/HTML-CSS-Popup/css/main.css

```diff
@@ -0,0 +1,187 @@
+* {
+    box-sizing: border-box;
+    -webkit-appearance: none;
+    outline-color: #1E7B0B;
+}
+
+body {
+    padding: 0;
+    margin: 0;
+    color: #000;
+    background-color: #E5E5E5;
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+}
+
+a {
+    color: inherit;
+    text-decoration: none;
+}
+
+ul {
+    padding: 0;
+    margin: 0;
+    list-style: none;
+}
+
+label {
+    cursor: pointer;
+}
+
+input[type="search"] {
+    width: 100%;
+    box-shadow: 0 1px 6px 0 rgba(32, 33, 36, .28);
+    box-sizing: border-box;
+    cursor: text;
+    font-size: 16px;
+    height: 44px;
+    margin: 0 auto;
+    border-radius: 22px;
+    background: transparent;
+    padding: 2px 22px;
+    border: 1px solid #1E7B0B;
+    background-color: #fff;
+}
+
+label[for="more"] {
+    display: block;
+    text-align: center;
+    padding: 10px 20px;
+    background-color: #1E7B0B;
+    color: #fff;
+}
+
+input[type="checkbox"] {
+    display: none;
+}
+
+/* input[type="search"]:focus {
+    background-color: #fff;
+} */
+
+.container {
+    display: flex;
+    justify-content: flex-end;
+    padding: 0 20px;
+}
+
+header {
+    padding: 5px 0;
+}
+
+#open-box:checked~.box {
+    display: block;
+}
+
+#more:checked~.list--two {
+    display: flex;
+}
+
+#more:checked~.list--two+a {
+    display: block;
+}
+
+#more:checked~label[for="more"] {
+    display: none;
+}
+
+.navigation__list {
+    display: flex;
+    align-items: center;
+}
+
+.navigation__list a:hover {
+    text-decoration: underline;
+}
+
+.navigation__list>li:not(:last-child) {
+    padding-right: 15px;
+}
+
+.user-avatar {
+    width: 32px;
+    height: 32px;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+/* box */
+.box {
+    width: 320px;
+    height: 300px;
+    overflow: auto;
+    background: #fff;
+    border: 1px solid #ccc;
+    border-color: rgba(0, 0, 0, .2);
+    color: #000;
+    -webkit-box-shadow: 0 2px 10px rgba(0, 0, 0, .2);
+    box-shadow: 0 2px 10px rgba(0, 0, 0, .2);
+    display: none;
+    position: absolute;
+    right: 10px;
+    margin-top: 20px;
+}
+
+.list {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 0 28px;
+}
+
+.list--two {
+    display: none;
+}
+
+.list+a {
+    padding: 10px 20px;
+    display: block;
+    text-align: center;
+    color: #1E7B0B;
+}
+
+.list--two+a {
+    display: none;
+}
+
+.list__item {
+    text-align: center;
+    width: 84px;
+    margin: 8px 1px;
+}
+
+.list__item a {
+    display: block;
+}
+
+.list__item a:hover {
+    outline: 1px solid #1E7B0B;
+}
+
+figure {
+    margin: 0;
+}
+
+figure img {
+    width: 44px;
+    height: 44px;
+    object-fit: contain;
+}
+
+.search-block {
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto;
+    max-width: 500px;
+    height: 90vh;
+    min-height: 300px;
+    align-items: center;
+    justify-content: center;
+}
+
+.logo {
+    max-width: 150px;
+    margin-bottom: 16px;
+}
```

Comment:
Add new line at end of files, it's common practice, if you don't use it, your files can be not correct concatenate. Check this in all files, please.